### PR TITLE
selenium added, spring server starts on tests, but with wrong db

### PR DESCRIPTION
**Pull request by alhopasi**
how to load test db for tests??

### CODE REVIEW

File: src/test/java/dtt/lukuvinkkikirjasto/demo/RunCucumberTest.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import io.cucumber.junit.Cucumber;
 import io.cucumber.junit.CucumberOptions;
 import io.cucumber.junit.CucumberOptions.SnippetType;
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
```
Comment (sokkanen) [inline src/test/java/dtt/lukuvinkkikirjasto/demo/RunCucumberTest.java]: @TestPropertySource(properties = { "url=./build/lukuvinkkitest.db"})

// Tämä saattaisi auttaa.

File: build.gradle [hunk 2]
```diff
@@ -19,11 +19,14 @@ sourceCompatibility = '1.8'
 
 project.ext {
 	cucumberVersion = '4.7.4'
+	seleniumVersion = '3.14.0'
+	fluentleniumVersion = '4.3.0'
+	springBootVersion = '2.1.3.RELEASE'
```
Comment (sokkanen) [inline build.gradle]: id 'org.springframework.boot' version '2.2.1.RELEASE' is already defined in the same directory

Comment (alhopasi) [reply build.gradle]: fixd

File: src/test/java/dtt/lukuvinkkikirjasto/demo/RunCucumberTest.java (hunk 1 above)
Comment (alhopasi) [reply src/test/java/dtt/lukuvinkkikirjasto/demo/RunCucumberTest.java]: did the other stuff that worked.

**Pull request merged**